specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -122,19 +122,28 @@ export function MergingDialog(): JSX.Element | null {
   const handleDismiss = (dismissedId: number) =>
     setIds(ids.filter((id) => id !== dismissedId).join(','));
 
+  const [mergeId, setMergeId] = React.useState<string | undefined>(undefined);
```

Comment:
you don't need this state in `<MergingDialog>` as you don't use it here
when I meant lifting the state, I meant lifting to `<Merging>` - because that's where it's used